docs-archive/apache-airflow-providers-airbyte/2.1.1/operators/airbyte.html

```diff
@@ -609,7 +609,7 @@ <h2>Using the Operator<a class="headerlink" href="#using-the-operator" title="Pe
 return the <code class="docutils literal notranslate"><span class="pre">job_id</span></code> that should be pass to the AirbyteSensor.</p>
 <p>An example using the synchronous way:</p>
 <div class="example-block-wrapper docutils container">
-<p class="example-header example-header--with-button"><span class="example-title">airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.py</span><a class="example-header-button viewcode-button reference internal" href="../_modules/airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.html"><span class="viewcode-link">[source]</span></a></p>
+<p class="example-header example-header--with-button"><span class="example-title">airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.py</span><a class="example-header-button viewcode-button reference internal" href="https://github.com/apache/airflow/tree/providers-airbyte/2.1.1/airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.py" target="_blank"><span class="viewcode-link">[source]</span></a></p>
```

Comment:
But this file exists https://airflow.apache.org/docs/apache-airflow-providers-airbyte/2.1.1/_modules/airflow/providers/airbyte/example_dags/example_airbyte_trigger_job.html

Why aren't we linking to it?